client/components/NavPane/NavPane.test.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import { shallow } from "enzyme"
+import NavPane from "./NavPane"
+
+const props = {
+  address: "",
+  location: {},
+  isFetching: true
+}
+
+describe("NavPane", () => {
+  it("renders the component", () => {
+    const wrapper = shallow(<NavPane />)
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  describe("onSubmit", () => {
+    it("calls getBoundaries", () => {
+      const getBoundaries = jest.fn()
+      const wrapper = shallow(
+        <NavPane.WrappedComponent {...props} getBoundaries={getBoundaries} />
```

Comment:
Learned something new about HOC components. Testing this way removes the need to find the button and simulate the click event.